### PR TITLE
Add  mobilenet_v2_1-4_244

### PR DESCRIPTION
**Pull request by pellegreene**

### CODE REVIEW

File: brainscore_vision/models/mobilenet_v2_1-4_224_pytorch/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+torchvision
+torch
```
Comment (mike-ferguson) [inline brainscore_vision/models/mobilenet_v2_1-4_224_pytorch/requirements.txt:2]: Are we leaving in torch in requirements.txt? Not sure what was decided @deirdre-k

**Review by mike-ferguson — APPROVED** [review]
lgtm - see comment about torch in `requirements.txt`

File: brainscore_vision/models/mobilenet_v2_1-4_224_pytorch/requirements.txt (hunk 1 above)
Comment (deirdre-k) [reply brainscore_vision/models/mobilenet_v2_1-4_224_pytorch/requirements.txt:2]: For now yeah. Once tests are migrated and the conda env approach in ironed out I think we should move it to a project-level dependency, but until then we want to leave it. Thanks for remembering!

Comment (mike-ferguson) [reply brainscore_vision/models/mobilenet_v2_1-4_224_pytorch/requirements.txt:2]: Sounds good - @pellegreene feel free to ignore my other PR comments regarding this then

**Pull request merged**